### PR TITLE
Adds recent AUV cruise and glider recovery cruise

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -362,3 +362,5 @@ EK20240815,Elakha,2024-08-15T15:00:00,2024-08-15T18:00:00,Endurance CSPP deploym
 KH20240821,Kahuna,2024-08-21T10:00:00,2024-08-22T19:30:00,Pioneer 20c glider deployments 
 KH20240917,Kahuna,2024-09-17T14:00:00,2024-09-18T18:00:00,Pioneer glider cp_564 recovery
 SKQ202415S,Sikuliaq,2024-10-04T20:40:00,2024-10-18T21:30:00,Endurance 21
+KH20241008,Kahuna,2024-10-08T09:00:00,2024-10-08T20:00:00,Pioneer glider cp_339 recovery
+VA20241012,Virginia,2024-10-12T06:00:00,2024-10-14T06:00:00,Pioneer 20d AUV deployment
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:366]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by rtravis-whoi — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
Updates made by me:
Sensor bulk load:
Addition of 5 new ADCPUP units (can verify in Asset_Tracking.xlsx
Addition of 6 TURBDX sensors (6 existing FLORT sensors w/ Turbd cal)
Node bulk load & eng bulk load:
Addition of new prawler 16035-05 (pic attached) - don't have documentation for the prawler sensors yet - will be added to SBL later
IMG_6545
IMG_6546

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -171,6 +171,7 @@ CGCON-CWFPCE-99103,A00285.1,Sensor,0,Coastal Wire-Following Profiler Controller,
 CGCON-CWFPCE-99104,A00303.1,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML12991-04,,20130510,0,,Oregon State University
 CGCON-CWFPCP-03501,,Sensor,0,Coastal Wire-Following Profiler Controller,Prawler Electronics,McLane,Prawler,ML16035-01,,,,,Woods Hole Oceanographic Institution
 CGCON-CWFPCP-03502,,Sensor,0,Coastal Wire-Following Profiler Controller,Prawler Electronics,McLane,Prawler,ML16035-02,,,,,Woods Hole Oceanographic Institution
+CGCON-CWFPCP-03505,,Sensor,0,Coastal Wire-Following Profiler Controller,Prawler Electronics,McLane,Prawler,ML16035-05,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:174]: looks good

File: bulk/node_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -69,6 +69,7 @@ CGVEH-CWFPCE-99103,A00285,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McL
 CGVEH-CWFPCE-99104,A00303,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,MMP-7,ML12991-04,,20130510,92259.49,210-3512,Oregon State University
 CGVEH-CWFPCP-03501,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,Prawler,ML16035-01,,,,,Woods Hole Oceanographic Institution
 CGVEH-CWFPCP-03502,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,Prawler,ML16035-02,,,,,Woods Hole Oceanographic Institution
+CGVEH-CWFPCP-03505,,Node,0,Coastal Wire-Following Profiler,WFP COASTAL,McLane,Prawler,ML16035-05,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/node_bulk_load-AssetRecord.csv:72]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -545,12 +545,17 @@ CGINS-ADCPTQ-01040,A04387,Sensor,0,ADCP Velocity Profiler (short range): ADCPT S
...
+CGINS-ADCPUP-00277,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400277,,20240212,20959,,Woods Hole Oceanographic Institution
+CGINS-ADCPUP-00280,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400280,,20240212,20959,,Woods Hole Oceanographic Institution
+CGINS-ADCPUP-00281,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400281,,20241022,20959,,Woods Hole Oceanographic Institution
+CGINS-ADCPUP-00282,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400282,,20241022,20959,,Woods Hole Oceanographic Institution
+CGINS-ADCPUP-00287,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400287,,20240212,20959,,Woods Hole Oceanographic Institution
+CGINS-ADCPUP-00291,,Sensor,0,ADCP Velocity Profiler (ultra-short range): ADCPU Series P,ADCPU SERIES P,Nortek,Aquadopp Profiler-S1VP,400291,,20241022,20959,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:558]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -2564,14 +2569,20 @@ CGINS-SPKIRJ-00265,N00560,Sensor,0,Spectral Irradiance: SPKIR Series J,SPKIR for
...
+CGINS-TURBDX-01218,,Sensor,0,Turbidity Sensor: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1218,,20140821,13013.52,same instrument as CGINS-FLORTD-01218 with 0-250 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01219,,Sensor,0,Turbidity Sensor: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1219,,20150911,13013.52,same instrument as CGINS-FLORTD-01219 with 0-250 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01295,,Sensor,0,Turbidity Sensor: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1295,,20150504,13403.46,same instrument as CGINS-FLORTD-01295 with 0-200 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01319,,Sensor,0,Turbidity Sensor: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1319,,20150717,13403.46,same instrument as CGINS-FLORTD-01319 with 0-250 NTU cal,Woods Hole Oceanographic Institution
+CGINS-TURBDX-01320,,Sensor,0,Turbidity Sensor: TURBD Series X,TURBD SERIES X (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1320,,20150717,13403.46,same instrument as CGINS-FLORTD-01320 with 0-250 NTU cal,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:2585]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:558]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 5 above)
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:2585]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**